manifests/monitoring/kube-prometheus.yaml

```diff
@@ -298,6 +298,35 @@ spec:
     {{ end }}
 ---
 apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: elasticsearch
+  namespace: monitoring
+spec:
+  name: elasticsearch.yaml
+  datasources:
+    - name: elasticsearch
+      type: elasticsearch
+      access: proxy
+      basicAuth: true
+      url: "https://{{.filebeat.elasticsearch.url}}:9200"
```

Comment:
I think this datasource will need to be guarded by an if statement for when filebeat/elasticsearch is not null.

The URL also shouldn't append :9200,  elastic can be server on  443/80